src/View/Components/Interaction/Dialog.php

```diff
@@ -37,6 +37,7 @@ public function personalization(): array
                 'wrapper' => 'mt-3 text-center sm:mt-5',
                 'title' => 'text-lg font-semibold leading-6 text-gray-700 dark:text-dark-200',
                 'content' => 'text-sm text-gray-500 dark:text-dark-300',
```

Comment:
Transform the `content` into an array with `base` and `wrapper`. If not `base`, use any other descriptive name.